changelogutils/reader_test.go

```diff
@@ -41,8 +40,22 @@ var _ = Describe("ReaderTest", func() {
 			file = changelogutils.ChangelogFile{
 				Entries: []*changelogutils.ChangelogEntry{&entry},
 			}
+			log  = "1.yaml"
+			repo = "testrepo"
+			sha  = "9065a9a84e286ea7f067f4fc240944b0a4d4c82a"
 		)
 
+		if os.Getenv("HAS_CLOUDBUILD_GITHUB_TOKEN") == "" {
+			log = "new-signature-manager.yaml"
+			repo = "reporting-client"
+			sha = "af5d207720ee6b548704b06bfa6631f9a2897294"
+			entry = changelogutils.ChangelogEntry{
+				Type:        changelogutils.NEW_FEATURE,
+				Description: "New signature manager implementation to be used in CLI clients that writes the signature to ~/.soloio",
+				IssueLink:   "https://github.com/solo-io/gloo/issues/1559",
+			}
+		}
```

Comment:
This is the general logic for replacing the private repo structure bits with things that work for everyone who isn't a member of solo-io.